reana_commons/config.py

```diff
@@ -133,3 +133,5 @@
 
 REANA_WORKFLOW_UMASK = 0o0002
 """Umask used for workflow worksapce."""
+
+K8S_DEFAULT_NAMESPACE = "DEFAULT"
```

Comment:
We should probably add a docstring explaining what it is and maybe even renaming the variable so we state clearly that this namespace is the namespace where all REANA Workflow Engines and Kubernetes jobs are going to be spawned in.

To help with the naming, this is closely related to Tibor's https://github.com/reanahub/reana-job-controller/issues/71#issuecomment-448928852, where he makes reference to the five compute layers of the REANA platform and this variable would affect layer 2, so something like `K8S_WORKFLOW_RUNTIME_DEFAULT_NAMESPACE`, what do you think?:)